File: b1-programming-basics-python/02b-conditional-statements-exercise/05_godzilla_vs_kong.py
budget = float(input())
cast_number = int(input())
cast_clothing_price = float(input())
set_price = 0.1 * budget
clothing_price = cast_number * cast_clothing_price

if cast_number > 150:
    clothing_price *= 0.9

total_cost = set_price + clothing_price
difference = abs(total_cost - budget)

if budget < total_cost:
    print(f"Not enough money!")
    print(f"Wingard needs {difference:.2f} leva more.")

else:
    print(f"Action!")
    print(f"Wingard starts filming with {difference:.2f} leva left.")

File: programming-basics-javascript/02b-conditional-statements-exercise/05-godzilla-vs-kong.js
function godzillaVsKong(input) {
  let movieBudget = Number(input[0]);
  let numberOfActors = Number(input[1]);
  let priceOfClothing = Number(input[2]);

  priceOfStage = movieBudget * 0.1;
  totalPriceOfClothing = numberOfActors * priceOfClothing;

  if (numberOfActors >= 150) {
    discount = totalPriceOfClothing * 0.1;
    totalPriceOfClothing -= discount;
  }

  totalPriceOfMoive = priceOfStage + totalPriceOfClothing;
  moneyAbsolute = Math.abs(movieBudget - totalPriceOfMoive).toFixed(2);

  if (movieBudget >= totalPriceOfMoive) {
    console.log(`Action!`);
    console.log(`Wingard starts filming with ${moneyAbsolute} leva left.`);
  } else {
    console.log(`Not enough money!`);
    console.log(`Wingard needs ${moneyAbsolute} leva more.`);
  }
}
